python/cudf/cudf/core/indexing.py

```diff
@@ -187,7 +193,7 @@ def _loc_to_iloc(self, arg):
             return indices_from_labels(self._sr, arg)
 
         else:
-            arg = Series(column.as_column(arg))
+            arg = cudf.core.series.Series(cudf.core.column.as_column(arg))
```

Comment:
Probably no need to change?

Reply:
Per Prem's suggestion, I changed it to handle issues with circular import 

```
cudf/core/indexing.py:15: in <module>
    from cudf.core.series import Series
cudf/core/series.py:44: in <module>
    from cudf.core.indexing import _SeriesIlocIndexer, _SeriesLocIndexer
E   ImportError: cannot import name '_SeriesIlocIndexer' from partially initialized module 'cudf.core.indexing' (most likely due to a circular import) (/home/skirui/home/skirui/cudf/python/cudf/cudf/core/indexing.py)
```